29th_first_semina/app/src/main/java/com/example/a29th_first_semina/SignInActivity.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.a29th_first_semina
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import com.example.a29th_first_semina.databinding.ActivitySigninBinding
+
+class SignInActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySigninBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySigninBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        initButtonClickEvent()
+
+    }
+
+    private fun initButtonClickEvent() {
+
+        clickLoginEvent()
+        clickSignUpEvent()
+    }
+
+    private fun moveHomeActivity() {
+        val intent = Intent(this, HomeActivity::class.java)
```

Comment:
위에도 그렇고 인라인으로 아래와 같이 적어주셔도 될 것 같습니다. 그리고 이정도 양의 함수면 굳이 함수 분리까지 할 필요 없고 initButtonClickEvent에 몰아도 될 것 같습니다.

그리고 intent라고 하면 어떤 것이 떠오르시나요? 변수명을 보고 어떤 역할을 하는 변수인지 알아볼 수 있게 적어주면 좋을 것 같습니다 :)
```suggestion
        startActivity(Intent(this, HomeActivity::class.java))
```

Reply:
아 걍 앱잼하면서 습관돼서ㅋㅋㅋㅋㅋㅋㅋㅋ